pkgs/tools/filesystems/rmount/default.nix

```diff
@@ -12,23 +11,24 @@ stdenv.mkDerivation rec {
     sha256 = "1wjmfvbsq3126z51f2ivj85cjmkrzdm2acqsiyqs57qga2g6w5p9";
   };
```

Comment:
With GitHub, it is preferred to use fetchFromGitHub, since it will make use of GitHub specific end-points to fetch the revision in a way that's (1) safe from their sometimes-changing archives (2) better-suited to fetch than plain git. Its use is pretty much like fetchgit, except that you pass `owner` and `repo` instead of `url`.